projects/1.18/assets/ecologics/ecologics/lang/zh_cn.json

```
@@ -0,0 +1,226 @@
+  "block.ecologics.walnut_beam": "核桃木支撑柱",
+  "block.ecologics.walnut_palisade": "核桃木实心栅栏",
+  "block.ecologics.walnut_seat": "核桃木板凳",
+  "block.ecologics.walnut_support": "核桃木支架",
+  "block.ecologics.azalea_log_bridge_middle": "杜鹃木桥",
+  "block.ecologics.flowering_azalea_log_bridge_middle": "杜鹃花木桥",
+  "block.ecologics.coconut_log_bridge_middle": "椰木桥",
+  "block.ecologics.walnut_log_bridge_middle": "核桃木桥",
+  "block.ecologics.rope_azalea_bridge": "杜鹃木索桥",
+  "block.ecologics.rope_flowering_azalea_bridge": "杜鹃花木索桥",
+  "block.ecologics.rope_coconut_bridge": "椰木索桥",
+  "block.ecologics.rope_walnut_bridge": "核桃木索桥",
+  "block.ecologics.azalea_bridge_pier": "杜鹃木桥梁支架",
+  "block.ecologics.flowering_azalea_bridge_pier": "杜鹃花木桥梁支架",
+  "block.ecologics.coconut_bridge_pier": "椰木桥梁支架",
+  "block.ecologics.walnut_bridge_pier": "核桃木桥梁支架",
+  "block.ecologics.azalea_log_bridge_stair": "杜鹃木桥梯",
+  "block.ecologics.flowering_azalea_log_bridge_stair": "杜鹃花木桥梯",
+  "block.ecologics.coconut_log_bridge_stair": "椰木桥梯",
+  "block.ecologics.walnut_log_bridge_stair": "核桃木桥梯",
+  "block.ecologics.azalea_rope_bridge_stair": "杜鹃木索桥梯",
+  "block.ecologics.flowering_azalea_rope_bridge_stair": "杜鹃花木索桥梯",
+  "block.ecologics.coconut_rope_bridge_stair": "椰木索桥梯",
+  "block.ecologics.walnut_rope_bridge_stair": "核桃木索桥梯",
+  "block.ecologics.azalea_rail_bridge": "杜鹃木铁轨桥",
+  "block.ecologics.flowering_azalea_rail_bridge": "杜鹃花木铁轨桥",
+  "block.ecologics.coconut_rail_bridge": "椰木铁轨桥",
+  "block.ecologics.walnut_rail_bridge": "核桃木铁轨桥",
+  "death.attack.coconut": "%1$s被掉落的椰子砸到了",
+  "death.attack.coconut.player": "%1$s在与%2$s战斗时被掉落的椰子砸到了",
+  "ecologics.subtitles.block.coconut.smash": "椰子：坠落",
+  "ecologics.subtitles.block.thin_ice.crack": "薄冰：破裂",
+  "ecologics.subtitles.entity.coconut_crab.ambient": "椰子蟹：嘶嘶声",
+  "ecologics.subtitles.entity.coconut_crab.death": "椰子蟹：死亡",
+  "ecologics.subtitles.entity.coconut_crab.hurt": "椰子蟹：受伤",
+  "ecologics.subtitles.entity.penguin.ambient": "企鹅：嘎嘎声",
+  "ecologics.subtitles.entity.penguin.death": "企鹅：死亡",
+  "ecologics.subtitles.entity.penguin.hurt": "企鹅：受伤",
+  "ecologics.subtitles.entity.squirrel.ambient": "松鼠：嗅探",
+  "ecologics.subtitles.entity.squirrel.death": "松鼠：死亡",
+  "ecologics.subtitles.entity.squirrel.hurt": "松鼠：受伤",
+  "ecologics.subtitles.entity.camel.ambient": "骆驼：吼叫",
+  "ecologics.subtitles.entity.camel.angry": "骆驼：怒吼",
+  "ecologics.subtitles.entity.camel.chest": "骆驼：装备箱子",
+  "ecologics.subtitles.entity.camel.death": "骆驼：死亡",
+  "ecologics.subtitles.entity.camel.eat": "骆驼：进食",
+  "ecologics.subtitles.entity.camel.hurt": "骆驼：受伤",
+  "ecologics.subtitles.entity.camel.step": "骆驼：脚步声",
+  "entity.ecologics.coconut_crab": "椰子蟹",
+  "entity.ecologics.camel": "骆驼",
+  "entity.ecologics.penguin": "企鹅",
+  "entity.ecologics.squirrel": "松鼠",
+  "entity.ecologics.boat": "船",
+  "effect.ecologics.slippery": "滑溜",
+  "effect.ecologics.slippery.description": "使受影响的实体像在冰上一样在所有方块上滑动。",
```

```suggestion
  "effect.ecologics.slippery.description": "使实体能在在所有方块上滑行，如同在冰上一样。",
```
Make affected entities slide on all blocks as if they were ice.